### PR TITLE
Update Hybrid RC Liftspeed

### PR DESCRIPTION
**Pull request by karst**
In this PR I have upped the lift speed of the RMC Hybrid to reflect the liftspeed of Lightning Rod, which should crest the lifthill with 13 mph, or 21 kmh respectively. I suspect the average lift speed to be about 11 mph / 17 kmh. 

https://youtu.be/vlWbtwIAnXE?t=156

With this I have also updated the ride music to use Rock style 4 by default.

### CODE REVIEW

File: src/openrct2/network/NetworkBase.cpp [hunk 1]
```diff
@@ -46,7 +46,7 @@ using namespace OpenRCT2;
 // It is used for making sure only compatible builds get connected, even within
 // single OpenRCT2 version.
 
-#define NETWORK_STREAM_VERSION "0"
+#define NETWORK_STREAM_VERSION "1"
```
Comment (spacek531) [inline src/openrct2/network/NetworkBase.cpp:49]: Can you rebase this?

Comment (karst) [reply src/openrct2/network/NetworkBase.cpp]: Done :)

**Review by spacek531 — APPROVED** [review]

**Comment by Gymnasiast** [comment]
Left this for far too long. Sorry about that.

I think it might be better not to do this, for two reasons:
- The Hybrid is already a very powerful roller coaster, having a 21 km/h chainlift (even if realistic) might upset balance further. 
- It now defaults to a music object that is not always available, which in practical terms means it will now default to a blank music style.

File: distribution/changelog.txt [hunk 2]
```diff
@@ -18,6 +18,7 @@
 - Change: [#7248] Small mini-maps are now centred in the map window.
 - Change: [#20240] Heavy snow and blizzards now make guests buy and use umbrellas.
 - Change: [#21214] Wacky Worlds and Time Twister’s scenario names now match their park names.
+- Change: [#21659] Up Hybrid Roller Coaster liftspeed to 11mph/17kmh.
```
Comment (AaronVanGeffen) [inline distribution/changelog.txt:21]: What does this mean? The default lift speed? Please clarify the line.

**Comment by karst** [comment]
>     * The Hybrid is already a very powerful roller coaster, having a 21 km/h chainlift (even if realistic) might upset balance further.

I have discussed this with Marcel and a few others, shouldn't impact the balance, in the grand scale of things it's not big of a change.

>     * It now defaults to a music object that is not always available, which in practical terms means it will now default to a blank music style.

That's true, but this is going to become an issue in the future in general. For OpenScenarios where RCT2 music will not be enabled by default but the OpenMusic objects instead.

Tho we should probably implement fallback for that if we do want to implement this and that might not be right for this PR.

File: distribution/changelog.txt (hunk 2 above)
Comment (karst) [inline distribution/changelog.txt:21]: > Up Hybrid Roller Coaster maximum liftspeed to 11mph/17kmh.

Comment (Gymnasiast) [reply distribution/changelog.txt:21]:
```suggestion
- Change: [#21659] Increase the Hybrid Roller Coaster’s maximum lift speed to 17 km/h (11 mph).
```

**Comment by Gymnasiast** [comment]
Very well, are you okay with doing just the lift speed increase and keeping the music style change out (at least until we have some sort of fallback)?

**Comment by karst** [comment]
Sounds like a plan

**Review by Gymnasiast — APPROVED** [review]

**Pull request merged**